components/splash/cards/whatIsFBCards.js

```diff
@@ -44,9 +44,10 @@ const Card = ({ icon, text, heading }) => (
       align='left'
       lineGap='1'
       lineColor='ocean'
-    >
-      <Icon name={icon} size='1.25rem' margin='.3rem .5rem .3rem 0' />
```

Comment:
i think the icon needs to be within the underlineHeading

Reply:
Fixed!